backend/controllers/sessionController.js

```diff
@@ -0,0 +1,39 @@
+const Session = require('../models/sessionModel');
+
+const sessionController = {};
+
+/**
+ * isLoggedIn - find the appropriate session for this request in the database, then
+ * verify whether or not the session is still valid.
+ */
+sessionController.isLoggedIn = (req, res, next) => {
+	// write code here
+	const ssid = req.cookies['ssid'];
+	if (!ssid) {
+		console.log('No SSID cookie, redirecting...');
+		return res.redirect('/signup');
```

Comment:
The redirect may introduce errors with react router... Would it be possible to respond with an error object?